templates/web/fixamingata/footer_extra.html

```diff
@@ -34,30 +34,30 @@ <h2 class="mysoc-footer__site-name">FixaMinGata</h2>
             <div class="col-sm-5">
                 <nav class="mysoc-footer__links">
                     <ul>
-                        <li><[% IF c.req.uri.path == '/' %]span[% ELSE %]a href="/"[% END %] class="report-a-problem-btn"
+                        <li><[% IF c.req.uri.path == '/' %]span[% ELSE %]a href="[% base %]/"[% END %] class="report-a-problem-btn"
```

Comment:
Can I ask why these are added? it feels like it might be a problem somewhere else that needs fixing?

Reply:
I don't remember why that was added, I'm afraid. 🤔 Let's remove it from this pull request!